### PR TITLE
Add v1.6 release note

### PR DESCRIPTION
**Pull request by emile-00**
## Info

- **Description**

  - Added the v1.6 release notes. We can either publish now to be in sync with the code or wait till docs are versioned and I'll go back and add links to the docs. Thanks team!

- **Notes**

  - [ Include any supplementary context or references here. ]

- **Related code PR**

  - [ Provide a link to the relevant code PR here, if applicable. ]

- **Related doc issue**
  
  Resolves [ Provide a link to the relevant doc issue here, if applicable. ]

## For reviewers

- **Preview**

  - [ Paste the preview link to the updated page(s) here. Edit this item after the preview site is ready. To find the updated pages, scroll down to locate and open the Amplify preview link and select the **dev** version of the documentation. ]

- **Key points**

  - [ Parts that may need revision or extra consideration. ]

## Before merging

- [ ] I have checked the doc site preview, and the updated parts look good.

- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`CharlieSYH`, `emile-00`, & `hengm3467`).

### CODE REVIEW

**Review by CharlieSYH — APPROVED** [review]
We should wait till the docs are versioned for 1.6 (in 1-2 days).

File: src/pages/release-notes.md [hunk 1]
```diff
@@ -9,6 +9,63 @@ slug: /release-notes
...
+    - **Breaking change**: Fixes the correctness of `SOME`, `ALL`, and `ANY` expressions. Drop and recreate any materialized views that use these expressions. [#14221](https://github.com/risingwavelabs/risingwave/pull/14221).
+    - Supports array subquery and \du command. [#14044](https://github.com/risingwavelabs/risingwave/pull/14044).
+    - Supports `SET PARALLELISM` clause for `ALTER` commands. [#14240](https://github.com/risingwavelabs/risingwave/pull/14240).
+- SQL commands:
+    - Technical preview feature: Supports `CREATE SINK INTO TABLE` Multiple sinks can use the same table as the destination. [#13185](https://github.com/risingwavelabs/risingwave/pull/13185), [#13659](https://github.com/risingwavelabs/risingwave/pull/13659).
+- SQL functions & operators:
+    - **Breaking change**: `0b10` is now interpreted as binary `10` instead of `0 as b10`. Integer literals can be given in hex `0x`, oct `0o` and bin `0b`. [#14262](https://github.com/risingwavelabs/risingwave/pull/14262).
+    - Supports interval type as input for `to_char()`. [#14071](https://github.com/risingwavelabs/risingwave/pull/14071).
```
Comment (WanYixian) [inline src/pages/release-notes.md]: As suggested by RC, we can add [#14080](https://github.com/risingwavelabs/risingwave/pull/14080) in SQL functions & operators。
The release note can be: Support NULL and fraction expression as direct arguments of ordered-set aggregate function. [#14080](https://github.com/risingwavelabs/risingwave/pull/14080).

**Review by WanYixian — APPROVED** [review]

File: src/pages/release-notes.md [hunk 2]
```diff
@@ -9,6 +9,63 @@ slug: /release-notes
...
+- Renames `scan.startup.timestamp_millis` to `scan.startup.timestamp.millis` for Kafka, Pulsar and NATS source. [#13656](https://github.com/risingwavelabs/risingwave/pull/13656).
+- Adds `properties.ssl.endpoint.identification.algorithm` parameter for Kafka source and sink.[#13990](https://github.com/risingwavelabs/risingwave/pull/13990).
+- Supports `FORMAT PLAIN ENCODE PROTOBUF` syntax for Kafka sink. [#12858](https://github.com/risingwavelabs/risingwave/pull/12858).
+- Supports GCS file source. [#13414](https://github.com/risingwavelabs/risingwave/pull/13414).
+- **Breaking change:** For ClickHouse sinks, `timestamptz` can be sinked to `DateTime64`. `timestamp` cannot be sinked and has to be converted to `timestamptz` first before being sinked. [#13672](https://github.com/risingwavelabs/risingwave/pull/13672).
+- For Elasticsearch sinks, the default es.type is set as `_doc` for Elasticsearch 6.x and 7.x, and is removed in Elasticsearch 8.x. [#14273](https://github.com/risingwavelabs/risingwave/pull/14273). Besides, RisingWave's Elasticsearch sink will now send JSONB as a JSON string, and Elasticsearch will convert it into an object.
```
Comment (ShanlanLi) [inline src/pages/release-notes.md]: Here I add some information:

..., and is removed in Elasticsearch 8.x. [#14273](https://github.com/risingwavelabs/risingwave/pull/14273). Besides, RisingWave's Elasticsearch sink will now send JSONB as a JSON string, and Elasticsearch will convert it into an object.

**Review by ShanlanLi — APPROVED** [review]
Rest LGTM.

**Comment by CharlieSYH** [comment]
@emile-00 1.6 is released.

**Pull request merged**